KoreBuild-dotnet/build/dotnet-install.sh

```diff
@@ -189,6 +189,9 @@ install_dotnet()
             say "Local Version: $localVersion"
 
             [ "$remoteHash" = "$localHash" ] && say "${green}You already have the latest version.${normal}" && return 0
+
+            dotnet_url="https://dotnetcli.blob.core.windows.net/dotnet/$CHANNEL/Binaries/$remoteVersion"
```

Comment:
Doesn't the url crafted on line 172 work? If you set it to Latest it should grab https://dotnetcli.blob.core.windows.net/dotnet/dev/Binaries/Latest/dotnet-osx-x64.latest.tar.gz which works?